### PR TITLE
[plugin-web-app-to-rest-api] Add ability to set checked attributes for resource checker

### PR DESCRIPTION
**Pull request by draker94**

### CODE REVIEW

File: vividus-plugin-web-app-to-rest-api/src/main/resources/properties/defaults/default.properties [hunk 1]
```diff
@@ -19,5 +19,6 @@ transformer.from-headless-crawling.respect-no-follow=false
 transformer.from-headless-crawling.respect-no-index=false
 
 resource-checker.uri-to-ignore-regex=
+resource-checker.attributes-to-check=href,src
```
Comment (uarlouski) [inline vividus-plugin-web-app-to-rest-api/src/main/resources/properties/defaults/default.properties:22]: do we really need this property? isn't it going to be always either href or src?

Comment (draker94) [reply vividus-plugin-web-app-to-rest-api/src/main/resources/properties/defaults/default.properties:22]: > do we really need this property? isn't it going to be always either href or src?

1. Sites can have their own attributes to specify, for example, image source 
2. We may want to check only links (href) but not page resources (src)

Comment (valfirst) [reply vividus-plugin-web-app-to-rest-api/src/main/resources/properties/defaults/default.properties:22]: yes, nope

File: vividus-plugin-web-app-to-rest-api/src/main/java/org/vividus/steps/integration/ResourceCheckSteps.java [hunk 2]
```diff
@@ -228,22 +232,25 @@ private static boolean isSchemaAllowed(URI uri)
...
 
-            String absUrl = element.absUrl(HREF_ATTR);
+            String absoluteUrl = element.absUrl(attributeWithValue);
             // For scripts e.g. href="javascript:alert('...');" the abs url will be empty
-            return absUrl.isEmpty() ? href : absUrl;
+            return absoluteUrl.isEmpty() ? hrefToCheck : absoluteUrl;
         }
-
-        return element.attr("src");
+        return attributeWithValue.isEmpty() ? StringUtils.EMPTY : element.attr(attributeWithValue);
```
Comment (valfirst) [inline vividus-plugin-web-app-to-rest-api/src/main/java/org/vividus/steps/integration/ResourceCheckSteps.java:253]:
```suggestion
        return attributeToCheck.map(element::attr).orElse(StringUtils.EMPTY);
```

File: vividus-plugin-web-app-to-rest-api/src/main/resources/properties/defaults/default.properties (hunk 1 above)
Comment (uarlouski) [reply vividus-plugin-web-app-to-rest-api/src/main/resources/properties/defaults/default.properties:22]: but for a its always gonna be href

Comment (valfirst) [reply vividus-plugin-web-app-to-rest-api/src/main/resources/properties/defaults/default.properties:22]: some platforms don't follow specifications

**Review by valfirst — APPROVED** [review]

File: vividus-plugin-web-app-to-rest-api/src/main/resources/properties/defaults/default.properties (hunk 1 above)
Comment (uarlouski) [reply vividus-plugin-web-app-to-rest-api/src/main/resources/properties/defaults/default.properties:22]: please provide markup example

**Pull request merged**